nautobot/extras/urls.py

```diff
@@ -296,4 +297,29 @@
         views.RelationshipAssociationDeleteView.as_view(),
         name="relationshipassociation_delete",
     ),
+    # Computed Fields
+    path("computed-fields/", views.ComputedFieldListView.as_view(), name="computedfield_list"),
+    path("computed-fields/add/", views.ComputedFieldEditView.as_view(), name="computedfield_add"),
+    path(
+        "computed-fields/delete/",
+        views.ComputedFieldBulkDeleteView.as_view(),
+        name="computedfield_bulk_delete",
+    ),
+    path("computed-fields/<uuid:pk>/", views.ComputedFieldView.as_view(), name="computedfield"),
```

Comment:
As mentioned in another comment, since we have a globally-unique `slug` field, consider using it instead of the UUID in these URLs?